plot/plot_lmbench

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
```

Comment:
thanks, one thing forget to say, help rename it to plot-lmbench to be align with other plot tools